modules/rostests/kmtests/ntos_cc/CcCopyRead_drv.c

```diff
@@ -36,6 +36,9 @@ FastIoRead(
     _Out_ PIO_STATUS_BLOCK IoStatus,
     _In_ PDEVICE_OBJECT DeviceObject)
 {
+    // Report whether this FastIO function is actually called or not.
+    ok(TRUE, "\n");
```

Comment:
Why does this need to just emit an extra newline?

Reply:
The newline is a safety and reminder. Text is useless in this usecase.
What would you prefer instead?